examples/src/changes.rs

```diff
@@ -0,0 +1,42 @@
+use near_primitives::{types::BlockReference, views::StateChangesRequestView};
+use serde_json::json;
+
+const STATUS_MSG_WASM_FILEPATH: &str = "./examples/res/status_message.wasm";
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox().await?;
+    let wasm = std::fs::read(STATUS_MSG_WASM_FILEPATH)?;
+    let contract = worker.dev_deploy(&wasm).await?;
+
+    let outcome = contract
+        .call("set_status")
+        .args_json(json!({
+            "message": "hello_world",
+        }))
+        .transact()
+        .await?;
+
+    let block_ref = {
+        let hash = near_primitives::hash::CryptoHash(outcome.outcome().block_hash.0);
+        BlockReference::BlockId(near_primitives::types::BlockId::Hash(hash))
+    };
+
+    let state_changes = {
+        StateChangesRequestView::ContractCodeChanges {
+            account_ids: vec![contract.id().clone()],
+        }
+    };
+
+    // NOTE: this API is under the "experimental" flag and no guarantees are given.
+    let res = worker.changes(block_ref, state_changes).await?;
```

Comment:
Same with this one and consistency with `Query` struct. We only need to supply `state_changes` while further methods can supply `block_ref` if it's needed with `Finality::Final` being the default when not supplied